src/app/exhibitor/_components/CompanySubmissionPopover.tsx

```diff
@@ -0,0 +1,172 @@
+"use client"
+import { sendToSlack } from "@/app/exhibitor/actions"
+import { Button } from "@/components/ui/button"
+import { env } from "@/env"
+import * as Popover from "@radix-ui/react-popover"
+import { Headset, X } from "lucide-react"
+import { useEffect, useRef, useState } from "react"
+import ReCAPTCHA from "react-google-recaptcha"
+
+export function CompanySubmissionPopover() {
+	const recaptcha = useRef<any>()
+	const [formData, setFormData] = useState({
+		name: "",
+		company: "",
+		email: "",
+		message: ""
+	})
+	const [isVerified, setIsVerified] = useState(false)
+	const [formedFilled, setFormFilled] = useState(false)
+	const [isOpen, setIsOpen] = useState(false)
+
+	useEffect(() => {
+		if (
+			formData.name !== "" &&
+			formData.email !== "" &&
+			formData.company !== "" &&
+			formData.message !== "" &&
+			isVerified
+		) {
+			setFormFilled(true)
+		} else {
+			setFormFilled(false)
+		}
+	}, [formData, isVerified])
+
+	const handleFieldChange = (event: { target: { name: any; value: any } }) => {
+		const key = event.target.name
+		const updatedFormValue = event.target.value
+		const newFormData = { ...formData, [key]: updatedFormValue }
+		setFormData(newFormData)
+	}
+
+	const handleVerify = (response: string | null) => {
+		if (response) {
+			setIsVerified(true)
+		} else {
+			setIsVerified(false)
+		}
+	}
+
+	const sendMessage = async () => {
+		const captchaValue = recaptcha.current.getValue()
+		if (!captchaValue) {
+			alert("Please verify the reCAPTCHA!")
+			return
+		}
+		await sendToSlack(formData).then(result => {
+			if (result.success) {
+				// Reset form fields
+				setFormData({
+					name: "",
+					email: "",
+					company: "",
+					message: ""
+				})
+				alert("Submitted! Our sale person will get in touch with you soon!")
+			} else {
+				alert("Submit failed! Please check your email format.")
+			}
+			setIsOpen(false)
+		})
+	}
+
+	return (
+		<div className="fixed bottom-0 mb-6 ml-auto mr-auto md:left-0 md:m-8 md:mx-auto md:ml-16">
```

Comment:
Make sure the trigger button always has the same padding to the left as to the right, on smaller screen sizes I get this

![CleanShot 2024-05-10 at 14 29 20@2x](https://github.com/armada-ths/armada.nu/assets/30767818/dedbcd21-04ec-45f2-b8a2-569892837c8e)